Makefile

```diff
@@ -11,8 +11,9 @@ BINDATA := $(shell find conf | sed 's/ /\\ /g')
 STYLESHEETS := $(wildcard public/less/index.less public/less/_*.less)
 JAVASCRIPTS :=
 
-LDFLAGS += -X "code.gitea.io/gitea/modules/setting.BuildTime=$(DATE)"
-LDFLAGS += -X "code.gitea.io/gitea/modules/setting.BuildGitHash=$(SHA)"
+VERSION = $(shell git describe --tags --always)
```

Comment:
Maybe even slap `--dirty` on it, so if you have a dirty tree you get `<tag>-dirty` instead
```
$ git describe --tags --abbrev=0 --dirty
v0.9.99-dirty
```